src/vs/workbench/parts/welcome/walkThrough/electron-browser/walkThroughPart.ts

```diff
@@ -517,7 +517,7 @@ export class WalkThroughPart extends BaseEditor {
 export const embeddedEditorBackground = registerColor('walkThrough.embeddedEditorBackground', { dark: null, light: null, hc: null }, localize('walkThrough.embeddedEditorBackground', 'Background color for the embedded editors on the Interactive Playground.'));
 
 registerThemingParticipant((theme, collector) => {
-	const color = getExtraColor(theme, embeddedEditorBackground, { dark: 'rgba(0, 0, 0, .4)', extra_dark: 'rgba(200, 235, 255, .064)', light: 'rgba(0,0,0,.08)', hc: null });
```

Comment:
Good for Light and Light+. Almost no contrast in Quiet Light and not a good match for Solarized Light.

Reply:
We're only testing for themes that we own (Dark and Light), so accessibility issues from third-party themes will not be fixed.

Reply:
These two themes are both built-in.

Reply:
The requirement was to only address our default themes (Dark and Light). Even though they come pre-installed, Quiet Light and Solarized are still third party themes and are not in scope.

@auchenberg, can you confirm?